doc/api/dns.md

```diff
@@ -353,15 +353,15 @@ be an object with the following properties:
 * `minttl`
 
 ```js
-{
+({
```

Comment:
I think readability should be our main concern in the docs. If a code block which isn't technically valid JS communicates more clearly than the valid-JS alternative, then we should keep using invalid JS and put a disable comment above it. In this case, I don't have a strong opinion on whether the parentheses increase readability.